rewards/service.go

```diff
@@ -0,0 +1,86 @@
+package rewards
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/brave-intl/bat-go/utils/clients/ratios"
+	appctx "github.com/brave-intl/bat-go/utils/context"
+	"github.com/brave-intl/bat-go/utils/logging"
+	srv "github.com/brave-intl/bat-go/utils/service"
+	"github.com/shopspring/decimal"
+)
+
+func init() {
+	// remove the quotes in json marshaling
+	decimal.MarshalJSONWithoutQuotes = true
```

Comment:
I would avoid having this be the default case and maybe do an explicit conversion to float instead. I think for this particular endpoint it doesn't matter much if we're losing precision but elsewhere it might ( e.g. if JS code parses the resulting json without quotes it will interpret it as a JS number )

Reply:
I started by using float64, then converted to decimal.  Didn't know how much the precision mattered, and it was constantly being downsampled.  Will change it back to use the float64 version.

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/407/commits/3f607368e26cefee09cdaec12ac7b5649b7793c7 and https://github.com/brave-intl/bat-go/pull/407/commits/7296fd298ff84c30996fd38bdfc6a2664088a532